.env.vagrant

```diff
@@ -17,6 +17,7 @@ SUPPORTED_MIME_TYPES='application/pdf,image/jpeg,image/png,application/vnd.openx
 MAX_NUMBER_OF_FILES=10
 
 FILE_TRANSFER_FEATURE=1
+ILLNESS_REASON_FEATURE=1
```

Comment:
Has variable also been added to the chs-config and docker development repos?

Reply:
not yet ;) but thanks for the reminder